scraper.rb

```diff
@@ -3,13 +3,17 @@
 # frozen_string_literal: true
 
 require 'pry'
+require 'pp'
 require 'require_all'
+require 'scraped_page_archive/open-uri'
 
 require_rel 'lib'
 
-page = MembersPage.new('http://house.gov.by/ru/deputies-ru/')
+page = MembersPage.new(
+  response: Scraped::Request.new(
+    url: 'http://house.gov.by/ru/deputies-ru/'
+  )
+                             .response(decorators: [AbsoluteLinks])
```

Comment:
With the current version of `Scraped`, decorators should be defined on the class — see https://github.com/everypolitician/scraped/pull/32 for more info. (Moving that should hopefully also help get rid of this quite odd formatting!)